packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -31,6 +32,15 @@ const { prefix } = settings;
  * @returns {*} JSX Object
  */
 const LightboxMediaViewer = ({ title, image, copy, ...modalProps }) => {
+  useEffect(() => {
+    (async () => {
+      const embedData = await VideoPlayerAPI.embedVideo(
+        '0_uka1msg4',
+        document.getElementById('kaltura_player')
+      );
+      console.log(embedData);
```

Comment:
Shouldn't be hard coding this in the component. This story should just be capturing the creation of the service, so we shouldn't include any changes to this component here.

Reply:
yeah, correct for testing purpose I have added @jeffchew